tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -1108,6 +1187,19 @@ Status MarkForCompilationPassImpl::FindCompilationCandidates() {
 
   VLOG(2) << "sorted_nodes.size() = " << sorted_nodes.size();
 
+  auto whitelist = GetWhitelist();
```

Comment:
nitpicking, but for a small value copied once I would just avoid the unique_ptr altogether